ui/app/services/store.js

```diff
@@ -138,13 +138,23 @@ export default class StoreService extends Store {
     return resp;
   }
 
+  forceUnload(modelName) {
+    // For some reason peeking the records before unloading ensures we
+    // don't have ghost models leftover
+    // eslint-disable-next-line no-console
+    console.info(`unloading ${this.peekAll(modelName).length} records`);
+    run(() => this.unloadAll(modelName));
+    return;
```

Comment:
Nit but this return seems unnecessary since it will return at the end of the function anyway.